docs/introduction-to-smart-contracts.rst

```diff
@@ -31,20 +31,19 @@ Storage Example
         }
     }
 
-The first line simply tells that the source code is written for
-Solidity version 0.4.0 or anything newer that does not break functionality
-(up to, but not including, version 0.7.0). This is to ensure that the
-contract is not compilable with a new (breaking) compiler version, where it could behave differently.
+The first line tells you that the source code is written for
+Solidity version 0.4.0, or a newer version of the language that does not break functionality, up to, but not including, version 0.7.0.
```

Comment:
Actually, re-reading it - this is not even correct. It includes 0.5.0 which does break functionality. So I think it should be

`Solidity version 0.4.0 or a newer version of the language up to, but not including version 0.7.0`

Reply:
Updated